server/views/temporary-accommodation/booking-search/results.njk

```diff
@@ -11,7 +11,7 @@
 
 {# TODO Move this HTML to the searchByCrnOrNameResult once HTML can be standardised between referrals and bookings #}
 {% set resultsHtml %}
-    {% if (not context.errors.prn) and ((response.data|length > 0) or (not crn)) %}
+    {% if response.data|length > 0 %}
```

Comment:
Same as above, I think `or not crn` should still be here

Reply:
I'll add it back for the moment while we improve the UX.